Formula/mupdf.rb

```diff
@@ -22,22 +23,52 @@ class Mupdf < Formula
 
   depends_on "pkg-config" => :build
   depends_on "freeglut"
+  depends_on "freetype"
+  depends_on "gumbo-parser"
+  depends_on "harfbuzz"
+  depends_on "jbig2dec"
+  depends_on "jpeg-turbo"
   depends_on "mesa"
+  depends_on "mujs"
+  depends_on "openjpeg"
```

Comment:
May try to finish #103436 first to clean up dependency tree using `jpeg-turbo` everywhere.